static/lang/re-en.json

```diff
@@ -2089,6 +2089,9 @@
                 },
                 "Forager": {
                     "Note": "While using Survival to Subsist, if you roll any result worse than a success, you get a success. You can also provide for multiple creatures."
+                },
+                "MysteryConduit":{
+                    "SpellItemAlterationDescription":"When you cast a spell of 5th rank or lower that has no duration, instead of spending a spell slot, you can advance the effects of your oracular curse in the same manner as when you cast a revelation spell. A spell cast in this way gains the cursebound trait."
```

Comment:
```suggestion
                    "Note": "When you cast a spell of 5th rank or lower that has no duration, instead of spending a spell slot, you can advance the effects of your oracular curse in the same manner as when you cast a revelation spell. A spell cast in this way gains the cursebound trait."
```

Reply:
Also move this under the Oracle section instead of under Feat